controller/resource_test.go

```diff
@@ -252,84 +342,85 @@ func TestReconcile(t *testing.T) {
 				ServiceName:          c.sutServiceName,
 				ServiceState: ServiceState{
 					ConsulECSTasks: c.tasks,
+					ACLPolicies:    beforePolicies,
 					ACLTokens:      beforeTokens,
 				},
 				Log: log,
 			}
 
-			err = serviceInfo.Reconcile()
+			err := serviceInfo.Reconcile()
 			require.NoError(t, err)
 
 			serviceStateLister := ServiceStateLister{
 				ConsulClient: consulClient,
+				Partition:    c.sutServiceName.Partition,
 				Log:          log,
 			}
 
-			aclTokens, err := serviceStateLister.fetchACLTokens()
+			aclState, err := serviceStateLister.fetchACLState()
 			require.NoError(t, err)
 
+			var policies []*api.ACLPolicy
 			var tokens []*api.ACLToken
-			for _, token := range aclTokens[c.sutServiceName] {
-				tokens = append(tokens, &api.ACLToken{
-					Description:       token.Description,
-					ServiceIdentities: token.ServiceIdentities,
-				})
+			if state, ok := aclState[c.sutServiceName]; ok {
+				for _, policy := range state.ACLPolicies {
+					fmt.Println(">>> policy", policy.Description)
```

Comment:
nit: Use `t.Logf`

Reply:
thanks for catching my hacky debug statements :grimacing:

I've removed them.